internal/templates/assisted-installer/template.go

```diff
@@ -182,7 +182,7 @@ metadata:
   name: "{{ .SpecialVars.CurrentNode.HostName }}"
   namespace: "{{ .Spec.ClusterName }}"
   annotations:
-    siteconfig.open-cluster-management.io/sync-wave: "1"
+    siteconfig.open-cluster-management.io/sync-wave: "3"
```

Comment:
This feels like it deserves a callout in the description or it's own commit given that it's fixing a very specific issue.

Reply:
Moved to its own commit.